tests/AuthenticatedUser.py

```diff
@@ -100,12 +100,12 @@ def testEditWithAllArguments(self):
 
     def testEmails(self):
         self.assertEqual(
-            self.user.get_emails(),
+            [item.email for item in self.user.get_emails()],
```

Comment:
You probably need to pull this out of the list comprehension so you check at least primary and hopefully the other two for the first element.